grouper/fe/templates/service.html

```diff
@@ -0,0 +1,318 @@
+{% extends "base.html" %}
```

Comment:
In general, any case where we're blanketly-duplicating template logic used elsewhere with no changes, we should turn that into a macro.